[7.0-Backported] web tree date search

fossoult opened the pull request.

fossoult committed:
[7.0-Backport] web_tree_date_search

fossoult committed:
[7.0-Backport] web_tree_date_search

elicoidal commented on web_tree_date_search/README.rst:
```diff
@@ -0,0 +1,40 @@
+    :alt: License
+
+Tree Date Search
+==============================
```
Hi @fossoult  thanks for your contribution
I will check mostly format and guidelines in your PR
Can you remove extra underline and adjust with sentence length?

elicoidal commented on web_tree_date_search/README.rst:
```diff
@@ -0,0 +1,40 @@
+Tree Date Search
+==============================
+
+This module allow you to easily add dates range search fields on top of List View rows.
```
s/allow/allows
s/on top of List View rows./on top of the tree view.

elicoidal commented on web_tree_date_search/README.rst:
```diff
@@ -0,0 +1,40 @@
+
+These dates fields can be used in combination with the Search window.
+
+How to do ?
```
# usage

elicoidal replied:
Add:
This module requires developer skills to be used.

elicoidal commented on web_tree_date_search/README.rst:
```diff
@@ -0,0 +1,40 @@
+.. image:: web_tree_date_search/static/src/img/demo.png
+.. image:: web_tree_date_search/static/src/img/demo1.png
+
+Credits
```
I miss
- the bug tracker section
- Try me on Runbot section

elicoidal commented on web_tree_date_search/README.rst:
```diff
@@ -0,0 +1,40 @@
+----------
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
```
https

elicoidal commented on web_tree_date_search/README.rst:
```diff
@@ -0,0 +1,40 @@
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
https

elicoidal commented on web_tree_date_search/__init__.py:
```diff
@@ -0,0 +1,18 @@
+##############################################################################
```
Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

fossoult committed:
OCA new guidelines
remove readme and add description on openerp.py
remove demo images and add new demo v7.0 image

fossoult committed:
OCA new guidelines
remove readme and add description on openerp.py
remove demo images and add new demo v7.0 image

fossoult commented:
@elicoidal  Should I remove the line 33 to pass the test ?

.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
   :alt: Try me on Runbot
   :target: https://runbot.odoo-community.org/runbot/162/7.0

elicoidal commented:
cc @pedrobaeza

fossoult commented:
I can update with " # noqa: skip 79 char because I can't change the URL"

fossoult committed:
Update __openerp__.py

fossoult committed:
Update __openerp__.py

fossoult commented:
Can you merge it plz ?

hbrunn commented:
@fossoult when backporting stuff, pick the original commits as explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-8.0#how with the necessary changes

rousseldenis commented on web_tree_date_search/__openerp__.py:
```diff
@@ -0,0 +1,86 @@
+        'static/src/js/search.js',
+    ],
+    'qweb': [
+        'static/src/xml/*.xml',
```
@fossoult Please name your file(s). Not very safe

rousseldenis reviewed (changes requested):
code review. Minor comment

rousseldenis reviewed (commented):
@fossoult Have you done all modifications requested by @elicoidal and @hbrunn ?

tfossoulw committed:
Add force date format on datetime field feature

tfossoulw committed:
Add force date format on datetime field feature

rousseldenis reviewed (commented):
@tfossoulw Can you check lint errors in javascript and solve them?

tfossoulw committed:
fix jslint warning

tfossoulw committed:
Merge branch '7.0-add_web_tree_date_search' of https://github.com/fossoult/web into 7.0-add_web_tree_date_search

# Conflicts:
#	web_tree_date_search/static/src/js/search.js

tfossoulw committed:
Fix js lint

tfossoulw committed:
Fix js lint

gurneyalex commented:
Hello @fossoult 

I'm having issues on runbot with this PR. Would you mind making a new commit and pushing

fossoult commented:
@gurneyalex I relaunch the runbot and it run without any problems. Can you tell me what you saw ?

rousseldenis reviewed (approved):
:+1: Seems ok. Flake8 error generated by web_warning_on_save module

tfossoulw committed:
Fix more jslint

gurneyalex commented:
ok then I consider this part of the docker migration of runbot solved. Thanks for you patience @fossoult
